app/commands/v2/discard_draft.rb

```diff
@@ -10,14 +10,13 @@ def call
 
         delete_supporting_objects
         delete_draft_from_database
-        delete_draft_from_draft_content_store(draft_path) if downstream
+        delete_draft_from_draft_content_store(draft_path)
```

Comment:
Would we also want the deletion of the draft to happen after the transaction completes?

I'm imagining the case where we've deleted the draft item from the draft content store, then this transaction fails (for whatever reason). We'd be in a state where we've deleted a draft but haven't written the live content item in its place. What do you think?